cpp/velox/operators/serializer/VeloxRowToColumnarConverter.cc

```diff
@@ -106,8 +106,8 @@ VectorPtr createFlatVector<TypeKind::HUGEINT>(
       int64_t offsetAndSize = *reinterpret_cast<int64_t*>(memoryAddress + offsets[pos] + fieldOffset);
       int32_t length = static_cast<int32_t>(offsetAndSize);
       int32_t wordoffset = static_cast<int32_t>(offsetAndSize >> 32);
-      uint8_t bytesValue[length];
-      memcpy(bytesValue, memoryAddress + offsets[pos] + wordoffset, length);
+      std::vector<uint8_t> bytesValue(length);
+      memcpy(bytesValue.data(), memoryAddress + offsets[pos] + wordoffset, length);
```

Comment:
offsetandsize is 8 bytes? right? if so we may needn't use a vector

Reply:
offsetAndSize is 8 bytes, but `length` is a variable extracted from `offsetAndSize`.